.github/workflows/ci-tests.yml

```diff
@@ -222,4 +222,4 @@ jobs:
       - name: Download current specification
         run: task ci:openapi:download
       - name: Ensure specification has not drifted
-        run: git diff --exit-code openapi.json
+        run: git diff --ignore-space-at-eol --exit-code openapi.json
```

Comment:
Why do we need to `ignore spaces at end of line` when comparing api spec. It should be generated by the same code and thereby be identical?

Reply:
We tried to explain in the commit comment adding the change:

> We'll allow spaces in eol to ignore diffs based on editor cleanup settings.